### PR TITLE
overrides: pin qemu-user-static-x86

### PR DESCRIPTION
**Pull request by nikita-dubrovskii**
qemu `9.2.0` fails with `segfault`. Pin it until https://github.com/coreos/fedora-coreos-tracker/issues/1846 gets fixed

### CODE REVIEW

**Review by jbtrystram — APPROVED** [review]
lgtm

File: manifest-lock.overrides.yaml [hunk 1]
```diff
@@ -29,3 +29,8 @@ packages:
     metadata:
       reason: https://github.com/coreos/fedora-coreos-tracker/issues/1836
       type: pin
+  qemu-user-static-x86:
+    evr: 2:9.1.1-2.fc42
+    metadata:
+      reason: https://github.com/coreos/fedora-coreos-tracker/issues/1846
+      type: pin
```
Comment (dustymabe) [inline manifest-lock.overrides.yaml:36]: do all architectures include this RPM? should it be an architecture specific override? Maybe it doesn't matter.

Comment (nikita-dubrovskii) [reply manifest-lock.overrides.yaml:36]: Just checked: all, but x86.

Comment (dustymabe) [reply manifest-lock.overrides.yaml:36]: then I guess if CI passes on this PR we should be good :)

**Pull request merged**